src/OSPSuite.Core/Domain/Builder/SpatialStructure.cs

```diff
@@ -154,40 +146,6 @@ public override void AcceptVisitor(IVisitor visitor)
             NeighborhoodsContainer.AcceptVisitor(visitor);
       }
 
-      /// <summary>
-      ///    Copy the references for first and second Neighbor from <paramref name="sourceNeighborhoodBuilder" /> to
-      ///    <paramref name="targetNeighborhoodBuilder" />.
-      /// </summary>
-      /// <param name="sourceNeighborhoodBuilder">The source neighborhood builder to copy neighbors from.</param>
-      /// <param name="targetNeighborhoodBuilder">The neighborhood builder to copy neighbors to.</param>
-      private void updateNeighborsReferences(INeighborhoodBuilder sourceNeighborhoodBuilder, INeighborhoodBuilder targetNeighborhoodBuilder)
-      {
-         var objectPathFactory = new ObjectPathFactory(new AliasCreator());
```

Comment:
all this god should be gone normally. Here we were trying to find a container based on the path etc.. well now let's just update the path